scripts/emerge/resolve_drs_uris.py

```diff
@@ -0,0 +1,80 @@
+"""This script resolves drs uris to gs paths."""
+
+import argparse
+import json
+from pydoc import resolve
+import requests
+from pprint import pprint
+
+from oauth2client.client import GoogleCredentials
+
+
+def get_access_token():
+    """Get access token."""
+
+    scopes = ["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"]
+    credentials = GoogleCredentials.get_application_default()
+    credentials = credentials.create_scoped(scopes)
+
+    return credentials.get_access_token().access_token
+
+
+def write_list_to_file(input_list, outfile_name):
+    """Write a list of values to a file."""
+
+    with open(outfile_name, "w") as outfile:
+        for item in input_list:
+            outfile.write(f"{item}\n")
+
+
+def resolve_drs_uri(drs_uuid):
+    """Resolve a drs uri to return the corresponding gs path."""
+
+    uri = f"https://data.terra.bio/ga4gh/drs/v1/objects/{drs_uuid}"
+
+    headers = {"Authorization": "Bearer " + get_access_token(), "accept": "*/*"}
+
+    response = requests.get(uri, headers=headers)
+    status_code = response.status_code
+
+    if status_code != 200:
+        return response.text
+
+    # get a list of the available access methods for a drs uri
+    avail_access_methods = response.json()["access_methods"]
+    # if there is an access method with "type" == "gs", get the gs url (other type options = https)
+    gs_url = [access_type["access_url"]["url"] for access_type in avail_access_methods if access_type["type"] == "gs"] # ["gs://"] or []
+
+    # if a gs url is not available, return "NA"
+    if not gs_url:
+        return "NA"
```

Comment:
is the string "NA" for sure what you want to be written to the file? will the output file only be human-read? if it's ever going to be used as input to another script, or uploaded to Terra, I might suggest setting this to `None` instead of the "NA" string.